### ISSUE TITLE
'Active' is being added to the 'wp_nav_menu' multiple times

### ISSUE THREAD

**Issue by alkymst** [roots/sage#493]
I have a possible fix, all though not fully tested.

``` php

add_filter('nav_menu_css_class', 'remove_active_nav_menu_css_class');
function remove_active_nav_menu_css_class($classes) {
    if(!empty($classes)) {  
        unset($classes);
        $classes = array('active');
    }
    return $classes;
}


```

**Comment by joelstransky** [comment]
I still have this issue in 7.0.2. My menu has a blog link and a category with categories as sub-menus. Both "blog" and "skills" gets 'active' when either one is selected. http://i.imgur.com/OxydCOa.png

**Comment by retlehs** [comment]
@joelstransky see @Foxaii's nav rejig branch on this repo / ref #986 

this is being addressed in 8.0.0

**Comment by joelstransky** [comment]
awesome thanks.
